packages/components/src/DateTimePickers/generator.js

```diff
@@ -6,6 +6,15 @@ import endOfMonth from 'date-fns/end_of_month';
 import format from 'date-fns/format';
 import setDay from 'date-fns/set_day';
 import startOfWeek from 'date-fns/start_of_week';
+import { getCurrentLanguage } from '../translate';
+
+export const locales = {
+	fr: require('date-fns/locale/fr'),
+	zh: require('date-fns/locale/zh_cn'),
```

Comment:
the issue with that part is that most app use zh as local for now we don't manage regions. if anyone has a proposal for that part feel free.

Reply:
This is not how we should do the things pour date-fns locales. Here it is quite strict, if we want to support another language, we have to change the code.
We decided to write our own date-fns locale, that use our i18next system. You can take a look at components/src/DateFnsLocale/locale.js.
It doesn't contain all translation, so if something is missing (like what you use here I think), please add it in the locale.
You can take example on any date-fns translation to see the api (https://github.com/date-fns/date-fns/tree/master/src/locale/fr)